scripts/convert_imgs_to_jpeg.py

```diff
@@ -1,6 +1,41 @@
 from wand.image import Image
 import os
+import json
+
+exceptions_file = open('photo_exceptions.json', 'rb')
+exceptions = json.load(exceptions_file)
+print exceptions
+
+def find_shift(name, axis):
```

Comment:
This should probably be get_shift, since it always returns a value. Find is for things that might fail because not-found.